Update auto-generated documentation

snoopycrimecop opened the pull request:
Repository: openmicroscopy/ome-documentation
Already up-to-date.



Generated by build [OMERO-DEV-latest-docs-autogen#1556](https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1556/).

----
--no-rebase

joshmoore commented:
Think @snoopycrimecop gets a 🍖 for PR `2000`

mtbc commented on omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh:
```diff
@@ -31,15 +31,15 @@ yum -y install openssl-devel bzip2-devel
 yum -y install ice-all-runtime
 
-pip install "zeroc-ice>3.5,<3.7"
+pip install https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
```
I've been using the same one for new-learning.

joshmoore replied:
Oh definitely. More just a question of whether or not we're ready to expose users to this. So far we've kept this pretty internal, e.g. it's not the default in prod playbooks, etc.

cc: @manics @jburel

joshmoore replied:
If we do want to go this route, https://github.com/ome/zeroc-ice-py-centos7/releases/latest/download/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl might be one idea to be more stable. Another would be to introduce a redirect via .htaccess:

```
[jamoore@idr0-slot3 wheel]$ cat .htaccess
Redirect 301 /ice/wheel/ https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
```

snoopycrimecop committed:
Update auto-generated documentation

Repository: openmicroscopy/ome-documentation
Already up-to-date.

Generated by OMERO-DEV-latest-docs-autogen#1558 (https://ci.openmicroscopy.org/job/OMERO-DEV-latest-docs-autogen/1558/)

joshmoore committed:
Manually add scripts section
